Makefile

```diff
@@ -30,6 +30,11 @@ vet: ## Apply go vet to all go files
 unit: # Run unit test
 	$(DOCKER_CMD) go test -race -cover ./cmd/... ./pkg/...
 
+.PHONY: sec
+sec: # Run security static analysis
+	go get -u github.com/securego/gosec/cmd/gosec
+	gosec -severity medium -confidence medium -exclude G304 -quiet ./...
```

Comment:
Would it be possible to run the command from within a container?

Reply:
It is possible. That's actually what was proposed for the installer: https://github.com/openshift/installer/pull/2126/files

Reply:
Can you update the PR so `gosec` is run inside a container?

Reply:
Done! thanks for the feedback